src/main/spec/internal-subscriptions-sync-api-spec.yaml

```diff
@@ -224,7 +224,6 @@ paths:
     parameters:
       - name: orgId
         in: query
-        required: true
```

Comment:
Note that the API spec for these endpoints were wrong:
- the orgId is not mandatory/required
- the azureAccountId/awsAccountId is required

After these changes, the aws and azure endpoints work the same.